source/broadband/webpa_replace.c

```diff
@@ -491,13 +509,16 @@ static int getWritableParams(char *paramName, char ***writableParams, int *param
         WalInfo("parameterName: %s, CompName : %s, dbusPath : %s\n", paramName, ppComponents[0]->componentName, ppComponents[0]->dbusPath);
         ret = CcspBaseIf_getParameterNames(bus_handle,ppComponents[0]->componentName, ppComponents[0]->dbusPath, paramName,1,&val_size,&parameterInfo);
         WalPrint("val_size : %d, ret : %d\n",val_size,ret);
-        if(ret == CCSP_SUCCESS && val_size > 0)
+        if(ret == CCSP_SUCCESS && val_size > 0 && isRbus != NULL)
         {
+	    rbusMode = (strncmp(isRbus, "true", strlen("true")) == 0) ? 1 : 0;	
```

Comment:
Can you move this check to line 502 just after GET and log an error when isRbus is NULL